### PR TITLE
Request for review

### PR DESCRIPTION
**Pull request by oracleot**
The aim of this project was to clone the current [Smashing Magazine](https://www.smashingmagazine.com/) landing page with focus on the UX of the website.

Cloning the website with a focus on UX was done by playing with the different colors on the website with varying degrees of the gray color.

The sections highlighted in this clone are:

- Page navigation
- Featured latest articles
- Latest posts
- Different types of Ads
- Person of the week
- Selected authors
- Smashing Magazine Guides
- Newsletter subscription box
- Footer containing several topics

### CODE REVIEW

File: assets/styles.css [hunk 1]
```diff
@@ -0,0 +1,576 @@
...
+  background-color: var(--gray-scale3);
+}
+
+.article2 {
+  background-color: var(--gray-scale4);
+}
+
+.article3 {
+  background-color: var(--gray-scale5);
+}
```
Comment (kitpao) [inline assets/styles.css:169]: - [x] You have a blank space under articles 1 and 3. Please fix this
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,537 @@
...
+          </p>
+          <a href="#" class="arrow">→</a>
+        </article>
+        <article class="post"></article>
+      </section>
+
+      <!-- Latest Posts -->
+      <section class="ad">
+        <article class="post hover">
+          <p class="days">6 Days To Go</p>
```
Comment (kitpao) [inline index.html]: - [x] You are missing a row of articles here
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,537 @@
...
+        <img
+          class="rotate"
+          src="https://res.cloudinary.com/indysigner/image/upload/v1542959380/shop/design-systems-hardcover-book.png"
+          alt=""
+        />
+      </section>
+
+      <!-- Our Guides -->
+      <section class="guides">
+        <p class="ninety-deg">Our Guides</p>
```
Comment (kitpao) [inline index.html]: - [ ] You are missing a section here, remember we are doing a visual heatmap and this missing section takes almost all the user's attention at that part
image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,537 @@
...
+          <p>A SMASHING GUIDE TO</p>
+          <h2>JavaScript & Frameworks</h2>
+          <p>
+            React, Vue, or perhaps the lightweight Alpine.js? Whether you are an
+            experienced JavaScript developer working with one of these
+            frameworks, or just starting to learn, our authors have created some
+            tutorials for you.
+          </p>
+          <a href="#">Explore JavaScript & Frameworks</a>
+        </section>
```
Comment (kitpao) [inline index.html]: - [x] Here your layout varies, in 1440px screen as required, the original only has 2 articles per row
image

image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,537 @@
...
+          <li><a href="#">Events</a></li>
+          <li><a href="#">Membership</a></li>
+          <li><a href="#">Jobs</a></li>
+          <li>
+            <a class="menu-link" href="#"
+              >Menu &nbsp;<img
+                style="width: 20px;"
+                src="https://d33wubrfki0l68.cloudfront.net/e3f5fe5fb4ec15257cc4137f13a0d5081b4ea5f8/aca41/images/icons/menu-cat.svg"
+                alt="cat"
+            /></a>
```
Comment (kitpao) [inline index.html]: - [x] For good coding practices, avoid using inline styling in your HTML, please add your styles in your CSS file

**Review by kitpao — CHANGES_REQUESTED** [review]
# 👋 Hello @oracleot and @ggotora 

🌟 Nice work guys, you did well, your code clean and organized. And the fact that your page has the nice hover effects of the original, makes it look professional 🏅 

You are missing some requirements, please, check them below in the inline comments 🔽 

### ♻️ Status: Your project requires some additional changes ♻️
➡️ Please request another review once you are ready 📝 
___
##### 👉 If you have any doubts you can contact me in slack as @kitpao

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__

**Review by kitpao — CHANGES_REQUESTED** [review]
# 👋 Hi again @oracleot and @ggotora 

⭐ You did a great job fixing the required issues 👏 

✏️ Please note that you missed only one requirement since it is part of the layout
- [ ] Missing design system section:
image

### ♻️ Status: Your project requires 1 additional change ♻️
➡️ Please request another review once you are ready 📝 
___
##### 👉 If you have any doubts you can contact me in slack as @kitpao

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__

**Comment by oracleot** [comment]
@kitpao kindly check well, we have the section already
Screenshot 2020-08-05 at 1 18 39 PM

**Review by kitpao — APPROVED** [review]
# 👋 Hi,

This is the strangest thing ever, I checked your project on Chrome and it is not displaying, changed to Firefox and it is there.
[Here is a recording of the screen](http://youtu.be/S-tPpjyBywA?hd=1)

Here is Chrome's screenshot, I can see the code, but it is not displaying
image

Since this is a browser issue, you don't have to worry,
You have complied with all required changes

### 👏 Your project has been approved 👏
### 🚀 You may merge to master and submit the project completition form 🚀

___
##### 👉 If you have any doubts you can contact me in slack as @kitpao

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__

**Pull request merged**